src/net/conn_plain.c

```diff
@@ -25,6 +27,24 @@ set_error(int err)
 #endif
 }
 
+static int
+get_error(void)
```

Comment:
```suggestion
get_socket_error(void)
```

Given that this is, at least on Windows, only used for Winsock-related errors.

Reply:
IMHO, the function itself is not restricted to socket-related errors, it is generic, and there is a corresponding function `set_error` above this one, so it makes sense to stick to the conventions already established by this.